.github/workflows/lint.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, 3.10]
+        python-version: [3.11]

```

Comment:
I think that you can remove the entire `strategy` section and use the single Python version directly in the "Set up Python" action in line 28 (replace `${{ matrix.python-version }}` with `3.11`. You don't need to include the name of the version in the name of the step, i.e., replace `Set up Python ${{ matrix.python-version }}` with just `Set up Python`.